lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1204,41 +1204,96 @@ def cmd_route_tabs(str, words)
   end
 
   def cmd_save_help
-    print_line "Usage: save"
+    print_line "Usage:"
     print_line
-    print_line "Save the active datastore contents to disk for automatic use across restarts of the console"
+    print_line "save:                       save the active module options to disk for automatic use across restarts of the console"
+    print_line "save -d / --delete:         delete the active module options from disk without refreshing current options"
+    print_line "save -r / --reload:         reload default options for the active module"
+    print_line "save -l / --load:           load the saved options for the active module"
+    print_line "save -D / --delete-all:     delete ALL datastore options from the config file"
```

Comment:
Not saying these options should be moved to a different command, just asking if anyone might know of a more intuitive place to put them? `save` is the command that handles interactions with the options file but UX wise the `save` command doing deletions is a sticking point